File: introduction/ITP1_10_C-Standard Deviation.java
import java.util.*;
public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int n, i;
    double s, m, sum;
    while((n = sc.nextInt()) != 0) {
      for(i = 0, m = 0, sum = 0; i < n; i++) {
        s = sc.nextDouble();
        m += s;
        sum += s*s;
      }
      m /= n;
      System.out.println(Math.sqrt(sum/n-m*m));
    }
  }
}

File: introduction/ITP1_10_C-Standard Deviation.cs
using System;
using System.Linq;
public class Program {
  public static void Main() {
    int n;
    while ((n = int.Parse(Console.ReadLine())) != 0) {
      double[] s = Console.ReadLine().Split(' ').Select(double.Parse).ToArray();
      double m = s.Average();
      Console.WriteLine(Math.Sqrt(s.Select(x => Math.Pow(x-m, 2)).Sum()/n));
    }
  }
}
